tests/test_project.py

```diff
@@ -562,6 +563,70 @@ def test_init_local_missing_west_yml_failure(repos_tmpdir):
         cmd(f'init -l "{zephyr_install_dir}"')
 
 
+def test_init_local_with_manifest_filename(repos_tmpdir):
+    # Test 'west init --mf -l' on a local repo
+
+    manifest = repos_tmpdir / 'repos' / 'zephyr'
+    workspace = repos_tmpdir / 'workspace'
+    zephyr_install_dir = workspace / 'zephyr'
+
+    # Do a local clone of manifest repo
+    clone(str(manifest), str(zephyr_install_dir))
+    os.rename(str(zephyr_install_dir / 'west.yml'),
+              str(zephyr_install_dir / 'project.yml'))
+
+    # fails because west.yml is missing
+    with pytest.raises(subprocess.CalledProcessError):
+        cmd(f'init -l "{zephyr_install_dir}"')
+
+    with open(zephyr_install_dir / 'west.yml', 'w') as f:
+        f.write('[')
+
+    # fails because of the syntax error
+    cwd = os.getcwd()
+
+    with pytest.raises(subprocess.CalledProcessError):
+        cmd(f'init -l "{zephyr_install_dir}"')
+        # init with a local manifest doesn't parse the file, so let's access it
+        workspace.chdir()
+        cmd('list')
+
+    os.chdir(cwd)
+    shutil.move(workspace / '.west', workspace / '.west-syntaxerror')
```

Comment:
Thanks for updating this! I'm glad to hear you're feeling better.

```suggestion
```

We are almost there. I just think this part of the test should be removed and then this is ready to go in. Sorry I was unclear about the intended behavior. What I meant was that since `west init -l` isn't parsing the file, there's no need for this part of the test.

You also need to rebase now.

If you can do those two things I am ready to approve. Thanks again :+1:

Reply:
well the reason I wrote that code is so I can verify that west is indeed using the correct manifest file. The syntax error is the way I detect if it's using the west.yml or the correct name that was specified.

Reply:
if you want me to apply your change anyway, just confirm that and I'll do it.